src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--days-in-month_29 calendar--start-day_sun">
```

Comment:
```suggestion
    <div class="calendar calendar--month-length--29 calendar--start-day--sun">
```
![image](https://user-images.githubusercontent.com/96409563/228746193-6eaddaa0-dc26-4e54-b2c4-495ca515464b.png)
![image](https://user-images.githubusercontent.com/96409563/228746255-29a5055d-fedd-4601-b9ab-f6a3128c1c4a.png)